modules/ROOT/pages/downgrade-runtime-fabric.adoc

```diff
@@ -0,0 +1,32 @@
+= Downgrading Runtime Fabric Versions
+
+If you need to downgrade Runtime Fabric version 2.6.x, you must:
+
+. Patch Runtime Fabric to the lower version
+. <<downgrade-rtf,Downgrade Runtime Fabric version>>
+
+== Downgrade Runtime Fabric Version
+
+To perform the downgrade of version, follow these steps:
+
+. Enable daemon-set log forwarding by calling the following API (bearer token required):
++
+----
+curl -X PATCH -H 'Authorization: Bearer <bearer-token-here>' -H 'Content-Type: application/json;charset=UTF-8' -H 'Accept: application/json, text/plain, */*' \
+https://anypoint.mulesoft.com/runtimefabric/api/organizations/<orgId>/fabrics/<fabricId>/logforwarding/appScoped?enabled=false
+----
+
+. Optionally, in the event am-log-forwarder daemon-set is not up and running, start the service by using either any of these options:
+
+* In the Anypoint Monitoring UI (*RuntimeFabrics --> Fabric --> LogForwarding*) check the option *Show logs in Anypoint Monitoring*.
+
+* Send an API request:
```

Comment:
```suggestion
* OR Send an API request:
```
There are 2 ways either via UI as stated above or by means of API

Reply:
Yes, that's why I indicated:
"start the service by using either any of these options:"

Reply:
Sorry my bad, I misread that.